src/core/server/rendering/rendering_service.tsx

```diff
@@ -151,7 +153,7 @@ export class RenderingService {
   }
 
   public checkUrlValid = async (url: string, configName?: string): Promise<boolean> => {
-    if (url.match(/\.(png|svg|gif)$/) === null) {
+    if (url.match(/\.(png|svg|gif|PNG|SVG|GIF)$/) === null) {
```

Comment:
good call can we add some tests to test the cases of where it has only PNG or SVG or GIF

Reply:
I thought about this too. Didn't end up doing this because the current official opensearch logo URL only ends with png and svg, didn't find any opensearch logo URL ends with gif, PNG, SVG, GIF. I could find some outside URL here for testing purpose, but I think we should not put any external URL in the repo? @kavilla